extras/kinesis-asl/src/main/scala/org/apache/spark/streaming/kinesis/KinesisBackedBlockRDD.scala

```diff
@@ -67,13 +74,13 @@ class KinesisBackedBlockRDDPartition(
 private[kinesis]
 class KinesisBackedBlockRDD(
     sc: SparkContext,
```

Comment:
Nit: Maybe it's better to add `@transient` to `sc`? Just for safety. We may change this class in future and the compiler will generate a field of `sc` if we use it in this class.